snippets/saved_replies.md

```diff
@@ -0,0 +1,103 @@
+
+If you think this is not purely a support question, feel free to comment in here or take the underlying issues to our [developer mailing list](https://groups.google.com/forum/#!forum/prometheus-developers).
+
+Once your questions have been answered, please add a short line pointing to relevant replies in case anyone stumbles here via a search engine in the future.
+
+All that being said, in your specific case, it might make sense to / please look at / our reasoning can be found at XXX
+
+
+# PRs
+
+
+## DCO
+
+Thanks for your PR. For legal reasons, we require that all commits are signed with a [DCO](https://developercertificate.org/) before we can merge them.
+
+This means that the last line of your commit message should read like:
+
+`Signed-Off-By: Your Name <your@email.address>`
+
+If you are using GitHub through the web interface, it's quickest to close this PR and open a new one with the appropriate line.
+
+If you are using Git on the command line, it is probably quickest to amend and force push. You can do that with
+````
+git commit --amend --signoff
+git push -f $remote $remote_branch_for_pr
+````
+
+As always, be careful when force-pushing.
+
+
+## Moratorium on integrations (SD & Alertmanager)
+
+Thanks for suggesting this change. Unfortunately, we are currently not accepting new integrations.
+
+We can only provide the stability and performance we want to provide if we can test integrations in an automated and scalable fashion. For this reason, we are suggesting people integrate with the help of our generic interfaces by either integrating natively or putting a rewriting proxy in the middle. We would be more than happy to list this on our integrations page.
+
+Even if existing integrations can not be tested in an automated fashion, we will not remove them for reasons of compability. This also means that any additions we take on, or any changes to existing integrations we make or accept, will mean maintaining and testing these until at least the next major version, realistically even beyond that.
+
+You are free to question this answer on our [developer mailing list](https://groups.google.com/forum/#!forum/prometheus-developers), but it's unlikely that you will get a different answer.
+
+
+# Issue or PR
+
+
+## Duplicate
+
+Thanks for reporting this. It seems we already have this issue tracked at XXX
+
+Thus, we are closing this one for now. If you think this was in error, don't hesitate to comment.
+
+
+## Non-trivial
+
+Thanks for this. This topic is not as easy as it looks; we actually already had some discussions and considerations about this topic, for more context please look here XXX
+
+
+## Out of scope
+
+Thank you for taking the time to write this out. Unfortunately, this is out of scope for what we are trying to achieve. You can find more information regarding our reasoning here XXX
+
+
+#### Philosophy
```

Comment:
Seems to me that Philosophy and Design decision can be grouped into the same answer. Made a suggestion below.